cobra/flux_analysis/loopless.py

```diff
@@ -1,6 +1,71 @@
 from ..core import Reaction, Metabolite
 from ..manipulation.modify import convert_to_irreversible
+import numpy as np
 from six import iteritems
+from cobra.util import nullspace, add_to_solver
+from sympy.core.singleton import S
+
+
+def add_loopless(model, zero_cutoff=1e-12):
+    """Add variables and constraints to make a model thermodynamically
+    feasible. This removes flux loops. The used formulation is described
+    in [1]_.
+
+    Parameters
+    ----------
+    model : a cobra model
+        The model to which to add the constraints.
+    zero_cutoff : positive float, optional
+        Cutoff used for null space. Coefficients with an absolute value smaller
+        than `zero_cutoff` are considered to be zero.
+
+    Returns
+    -------
+    Nothing
+
+    References
+    ----------
+    .. [1] Elimination of thermodynamically infeasible loops in steady-state
+       metabolic models.
+       Schellenberger J, Lewis NE, Palsson BO.
```

Comment:
I think in this case it's not that critical since Palsson is cited like that pretty often. If it would in actual code I would agree. But really have no preference. I can add the `u"""` if you would like.